src/utils/generateRandomColor.tsx

```diff
@@ -0,0 +1,11 @@
+function generateRandomColors(numColors:number):string[] {
+    const colors = [];
+    for (let i = 0; i < numColors; i++) {
+
+        const color = "#" + Math.floor(Math.random()*16777215).toString(16);
+        colors.push(color);
+    }
+    return colors;
+}
```

Comment:
can you add filter for this function which remove black or white shades of colors and replace it for other colors?